ee/maintained-apps/ingesters/homebrew/external_refs/main.go

```diff
@@ -0,0 +1,5 @@
+package externalrefs
+
+var Funcs = map[string]func(...interface{}) (string, error){
+	"MicrosoftVersionFromReleaseNotes": MicrosoftVersionFromReleaseNotes,
+}
```

Comment:
Trying to set this up for more version overrides in a generic way. And really overrides for any other attributes with more complicated logic. This is an effort not to litter the `ingestor.go` file with a bunch of random methods.

Reply:
Yep, this makes sense. Probably want to tweak the form a little (see my later comment for ideas).